standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1600,7 +1602,9 @@ public static ConfVars getMetaConf(String name) {
     // Deprecated Hive values that we are keeping for backwards compatibility.
     @Deprecated
     HIVE_CODAHALE_METRICS_REPORTER_CLASSES("hive.service.metrics.codahale.reporter.classes",
-        "hive.service.metrics.codahale.reporter.classes", "",
+        "hive.service.metrics.codahale.reporter.classes",
+        "org.apache.hadoop.hive.common.metrics.metrics2.JsonFileMetricsReporter, " +
+        "org.apache.hadoop.hive.common.metrics.metrics2.JmxMetricsReporter",
```

Comment:
What does this do? Especially since the config is deprecated?

Reply:
This param is used to initialize the CodahaleMetricsReporter classes. In some unit test, we are using it, and since the unit test were moved from the `hive-common` module to the `standalone-metastore-common` module, I had to add these values to the `MetastoreConf` to keep backward compatibility.

Reply:
I see, thanks for explaining!